runtime/lua/vim/lsp/buf.lua

```diff
@@ -297,26 +297,30 @@ local function pick_call_hierarchy_item(call_hierarchy_items)
   return choice
 end
 
+local function call_hierarchy(method)
+  local params = util.make_position_params()
+  request('textDocument/prepareCallHierarchy', params, function(err_message, _, result)
```

Comment:
minor nit, can you change the callback to use `err` instead of `err_message` like in https://github.com/neovim/neovim/blob/91cdc11984465287539ed8f63716bb371f440ea1/runtime/lua/vim/lsp/handlers.lua#L161

`err_message` is also the name of a function we use for error handling: 

https://github.com/neovim/neovim/blob/91cdc11984465287539ed8f63716bb371f440ea1/runtime/lua/vim/lsp/handlers.lua#L15-L18